Modules/Core/Common/wrapping/itkImageToImageFilterB.wrap

```diff
@@ -32,13 +32,18 @@ itk_wrap_class("itk::ImageToImageFilter" POINTER)
   itk_wrap_image_filter_combinations("${WRAP_ITK_COMPLEX_REAL}" "${WRAP_ITK_SCALAR}")
 
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
-    foreach(t ${WRAP_ITK_SCALAR})
-      # scalar <-> SymmetricSecondRankTensor
-      itk_wrap_template("${ITKM_I${t}${d}}${ITKM_ISSRT${ITKM_D}${d}${d}}" "${ITKT_I${t}${d}}, ${ITKT_ISSRT${ITKM_D}${d}${d}}")
```

Comment:
@tbirdso we were always wrapping for `double` before.

Reply:
Thanks for taking a look @thewtex, it didn't occur to me that we're not wrapping for `double` by default.